src/Components/Navbar.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { IoLogoDesignernews } from "react-icons/io";
+import { TiArrowBack } from "react-icons/ti";
+import { FaBars } from "react-icons/fa";
+
+const Navbar = () => {
+  const [show, setShow] = useState<boolean>(false);
+
+  const handleClick = () => {
+    setShow(!show);
```

Comment:
there is a way to do this
setShow(prevState => !prevState)
do read about it and let me know

Reply:
Done